.homeconfig

```diff
@@ -55,7 +55,7 @@ export HOME_POSTGRES_CFLAGS=""
 export HOME_POSTGRES_LDFLAGS=""
 
 # Custom options for Postgres ./configure
-export HOME_POSTGRES_CONFIGURE=""
+export HOME_POSTGRES_CONFIGURE="--enable-tap-tests"
```

Comment:
The answer to that is "force it in your own private profile". This code is intentional.